src/db-service.ts

```diff
@@ -28,7 +30,7 @@ export class FileUploadLimiterDbService {
           WHERE item.type = ${itemType} AND item.creator = ${memberId}`,
         )
         // sum up
-        .then(({ rows }) => parseInt(rows[0].sum, DECIMAL))
+        .then(({ rows }) => parseInt((rows[0] as { sum: string }).sum, DECIMAL))
```

Comment:
Does this not type-check anymore? Would it make sense to write it as `rows[0].sum as string` if that's the case? Not sure what's best